tests/manage/mcg/test_namespace.py

```diff
@@ -30,6 +31,199 @@ class TestNamespace(MCGTest):
     # is resolved
     DEFAULT_REGION = "us-east-2"
 
+    @skipif_ocs_version("<4.7")
+    @tier1
+    @pytest.mark.parametrize(
+        argnames="nss_tup",
+        argvalues=[
+            pytest.param(("oc", {"aws": [(1, "eu-central-1")]})),
```

Comment:
At the time being, yeah. It's more about testing AWS, rather than any particular region. We also have it hardcoded in a few other places (`test_multicloud`).
Solving it should be a part of the general solution of #3338

Reply:
Ack